sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations_base.py

```diff
@@ -345,7 +345,9 @@ def begin_update(
             system_datastores_auth_mode=kwargs.get(
                 "system_datastores_auth_mode", workspace.system_datastores_auth_mode
             ),
-            allow_roleassignment_on_rg=kwargs.get("allow_roleassignment_on_rg", workspace.allow_roleassignment_on_rg),
+            allow_role_assignment_on_rg=kwargs.get(
+                "allow_roleassignment_on_rg", workspace.allow_roleassignment_on_rg
```

Comment:
just want to confirm this is the correct format not allow_role_assignment_on_rg

Reply:
yes, it is correct.

`allow_role_assignment_on_rg` is coming from generating the _restclient code based on the swagger. (due to REST property casing "allowRoleAssignmentOnRG"). `WorkspaceUpdateParameters` comes from `azure.ai.ml._restclient.v2024_07_01_preview.models`

but in the SDK object itself we have `allow_roleassignment_on_rg` since that was what was added in [PR 35939](https://github.com/Azure/azure-sdk-for-python/pull/35939/files) and was already in a public released SDK so I couldnt match them :(